tool/tctl/common/status_command.go

```diff
@@ -160,5 +163,13 @@ func (c *StatusCommand) Status(ctx context.Context, client auth.ClientI) error {
 		}
 		fmt.Print(view())
 	}
+
+	if err := common.ShowClusterAlerts(ctx, client, os.Stdout, map[string]string{
+		types.AlertOnLogin: "yes",
+	}, map[string]string{
+		types.AlertLicenseExpired: "yes",
```

Comment:
Am I missing something, or will this actually not display license alerts? Since these are "ignore" labels?

Reply:
The intention here was to ignore the already expired cluster alerts since they will be displayed on every command and this prevents duplication.